### ISSUE TITLE
.version() and .igv_version() methods raise a FileNotFoundError

### ISSUE THREAD

**Issue by safay** [igvteam/igv-notebook#14]
Package version: igv-notebook==0.4.2, installed with `pip`
Python version: 3.9.15
OS: Ubuntu 18.04.6 LTS

Executing `igv_notebook.version()` as per the documentation raises an error:
```
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
Cell In[7], line 1
----> 1 igv_notebook.version()

File /opt/tljh/user/lib/python3.9/site-packages/igv_notebook/version.py:3, in version()
      2 def version():
----> 3     with open('VERSION') as version_file:
      4        return version_file.read().strip()

FileNotFoundError: [Errno 2] No such file or directory: 'VERSION'
```

A similar error is raised for the `.igv_version()` method.

Perhaps this is related to the fact that this is not a dev (`-e`) install, and it's just the README that needs updating?

**Comment by jrobinso** [comment]
That looks like a packaging bug.  Thanks for the report, I'll look into it.

**Comment by jrobinso** [comment]
This is now fixed in release 0.4.3
